js-core/week-04/lesson.md

```diff
@@ -18,26 +21,37 @@ This class is based on the
 
 ## Getting Started
 
-1. Download and install the latest Current version of NodeJS from
-   [here](https://nodejs.org/en/download/current/)
-
-To test that it was installed and running properly, go to your terminal and run
-the command: `node -v` You should get the node version printed on your terminal,
-for example, `v7.3.0`
-
-1. Fork the
+1. Download and install the latest Current version of NodeJS.
```

Comment:
I updated the link to point at download instead of download/current. This has the effect of linking students to the LTS (recommended) version, currently 8.x.x, rather than latest, which risks being on the verge of deprecation when installed